api/routes/auth.js

```diff
@@ -0,0 +1,26 @@
+// routes/auth.js
+import { Router } from 'express';
+const router = Router();
+
+router.get('/auth/check', (req, res) => {
+  console.log('--- AUTH CHECK CALLED ---');
```

Comment:
Do we need these console.logs for production? We don't want to pollute the console for clients with irrelevant data.